.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         cargo check --no-default-features --features async-std
         cargo check --no-default-features --features async-std,sparse
         cargo check --no-default-features --features async-std,sparse,cache
+        cargo test --no-default-features --features tokio,shared-core
```

Comment:
Any reason you aren't adding also async-std replication tests here too, instead of these tokio-only tests?

Reply:
I've added the `async-std` tests.